api/commands/init.ts

```diff
@@ -18,14 +18,21 @@ export interface InitResult {
   installedNpmPackages: boolean;
 }
 
+export interface InitOptions {
+  skipInstall?: boolean,
+  includeSchedules?: boolean
+}
+
 export async function init(
   projectDir: string,
   projectConfig: dataform.IProjectConfig,
-  skipInstall?: boolean
+  options: InitOptions
```

Comment:
I'd make this optional, and you can set a default value like `options?: InitOptions = {}`

Reply:
it seems you can't have both initial value and optional flag. So I gave initial value.